prymer/primer3/primer3_parameters.py

```diff
@@ -219,6 +219,7 @@ def to_input_tags(self) -> dict[Primer3InputTag, Any]:
             Primer3InputTag.PRIMER_INTERNAL_MAX_GC: self.probe_gcs.max,
             Primer3InputTag.PRIMER_INTERNAL_MAX_POLY_X: self.probe_max_polyX,
             Primer3InputTag.PRIMER_INTERNAL_MAX_NS_ACCEPTED: self.probe_max_Ns,
+            Primer3InputTag.PRIMER_NUM_RETURN: self.number_probes_return
```

Comment:
**note** `probe_max_dinuc_bases` is also not represented in the input tag mapping

Reply:
This one is actually on purpose - primer3 doesn't do dinuc checking.  It's definitely slightly confusing that we then supply the parameter is something that makes it look like it's going to primer3, but the check is done separately after primer3.